packages/react/src/NavList/NavList.stories.tsx

```diff
@@ -246,4 +247,26 @@ export const WithGroup = () => (
   </PageLayout>
 )
 
+export const WithTrailingAction = () => {
+  return (
+    <NavList>
+      <NavList.Item>
+        <NavList.LeadingVisual>
+          <FileDirectoryIcon />
+        </NavList.LeadingVisual>
+        Item 1
+        <NavList.TrailingAction label="Expand sidebar" icon={ArrowLeftIcon} />
+      </NavList.Item>
+      <NavList.Item>
+        Item 2
+        <NavList.TrailingAction as="a" href="#" label="Some action" icon={ArrowRightIcon} />
+      </NavList.Item>
+      <NavList.Item>
+        Item 3
+        <NavList.TrailingAction label="Another action" icon={BookIcon} />
+      </NavList.Item>
+    </NavList>
```

Comment:
Non-blocking: Should we wrap this within `<PageLayout>` like the other stories, so the styling is consistent? 🤔